### DISCUSSION TITLE
Romanian language

### DISCUSSION THREAD

**Discussion by 18rrs** [dckiller51/bodymiscale#107] [General]
Hy guys, i dont know hot to use github so i dont know where to put the romanian translation for this integration. I leave it here, sorry.
[ro.zip](https://github.com/dckiller51/bodymiscale/files/9076536/ro.zip)

**Comment by dckiller51** [comment]
hello, I integrate it in the week. Thank you for your contribution. If you wish, you can also translate the card.

**Reply by 18rrs** [reply]
sure.. what's the files name?

**Reply by dckiller51** [reply]
https://github.com/dckiller51/lovelace-body-miscale-card/tree/main/src/localize/languages

**Reply by 18rrs** [reply]
here it is.
[ro.zip](https://github.com/dckiller51/bodymiscale/files/9144801/ro.zip)

**Reply by dckiller51** [reply]
> here it is. [ro.zip](https://github.com/dckiller51/bodymiscale/files/9144801/ro.zip)

Hello, I have published the maj of the card (5.9.1).
